### PR TITLE
MAINT-25824 : CLV document with white space Wrong Display

### PR DESCRIPTION
**Pull request by Rdidmohamedelhadi**
**Problem :** CLV document with white space not  display.
**Solution :** Encode the `url` to display correctly.

### CODE REVIEW

File: ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-artifacts/application-templates/content-list-viewer/list/Documents.gtmpl [hunk 1]
```diff
@@ -72,7 +72,8 @@
 								
 								def itemLink = uicomponent.getURL(viewNode);			
 								def itemWebdavLink = uicomponent.getWebdavURL(viewNode);
-								uicomponent.setDateTimeFormat("dd/MM/yyyy");			
+								itemWebdavLink = uicomponent.encodeURI(itemWebdavLink);
```
Comment (rdenarie) [inline ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-artifacts/application-templates/content-list-viewer/list/Documents.gtmpl:75]: Do you think it is possible to do that without modifying the template ? 
If you modify the template, we need to use an UpgradePlugin for that. If you do the encoreURI in getWebdavURL, we dont have to change the gtmpl and no UP necessary

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-artifacts/application-templates/content-list-viewer/list/Documents.gtmpl (hunk 1 above)
Comment (Rdidmohamedelhadi) [reply ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-artifacts/application-templates/content-list-viewer/list/Documents.gtmpl]: Done !

File: core/webui-presentation/src/main/java/org/exoplatform/wcm/webui/clv/UICLVPresentation.java [hunk 2]
```diff
@@ -489,7 +491,7 @@ public String getWebdavURL(Node node) throws Exception {
           + workspace + node.getPath();
     }
 
-    return friendlyService.getFriendlyUri(link);
+    return friendlyService.getFriendlyUri(encodeURI(link));
```
Comment (rdenarie) [inline core/webui-presentation/src/main/java/org/exoplatform/wcm/webui/clv/UICLVPresentation.java:494]: Should it be :
```suggestion
    return encodeURI(friendlyService.getFriendlyUri(link));

```

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-artifacts/application-templates/content-list-viewer/list/Documents.gtmpl [hunk 3]
```diff
@@ -72,7 +72,7 @@
 								
 								def itemLink = uicomponent.getURL(viewNode);			
 								def itemWebdavLink = uicomponent.getWebdavURL(viewNode);
-								uicomponent.setDateTimeFormat("dd/MM/yyyy");
```
Comment (rdenarie) [inline ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-artifacts/application-templates/content-list-viewer/list/Documents.gtmpl:75]: you should not have a modification in this file;

**Review by rdenarie — CHANGES_REQUESTED** [review]

**Review by rdenarie — APPROVED** [review]

**Pull request merged**